src/components/Checkout/Checkout.jsx

```diff
@@ -106,162 +105,156 @@ export const Checkout = () => {
       updatedData.orderItems = orderItems;
 
       // Log to check data in the console and create order with the updated data
-      console.log(updatedData);
       await placeOrder(updatedData);
     } catch (error) {
-      console.error('Error submitting form:', error);
       toast.error('An error occurred while placing the order.');
     }
   };
 
+  // If the cart is empty, redirect to the homepage.
+  if (!cartItems?.length) {
```

Comment:
We should implement this check to block scenarios where users attempt to access the checkout page without any items in their cart and try to place orders.

Reply:
all good